de1plus/bluetooth.tcl

```diff
@@ -682,6 +682,40 @@ proc decentscale_tare {} {
 	userdata_append "decentscale : tare" [list ble write $::de1(scale_device_handle) $::de1(suuid_decentscale) $::sinstance($::de1(suuid_decentscale)) $::de1(cuuid_decentscale_write) $::cinstance($::de1(cuuid_decentscale_write)) $tare] 0
 }
 
+### Meater
+
+proc meater_enable_notifications {} {
+	if {$::de1(thermometer_device_handle) == 0 || $::settings(thermometer_bluetooth_type) != "apptionmeater"} {
+		return
+	}
+
+	if {[ifexists ::sinstance($::de1(suuid_decentscale))] == ""} {
+		msg "meater not connected, cannot enable weight notifications"
+		return
+	}
+
+	userdata_append "enable meater notifications" [list ble enable $::de1(thermometer_device_handle) $::de1(suuid_meater) $::sinstance($::de1(suuid_meater)) $::de1(cuuid_meater) $::cinstance($::de1(cuuid_meater))] 1
+}
+
+proc meater_parse_response { value } {
+	binary scan $value a2 raw
+	set temp [expr $raw / 16]
+
+	msg "Meater temperature is $temp"
```

Comment:
Hopefully PR #114 (Logging for v1.35) will be merged soon. 

Would it be reasonable to put the MEATER-specific code into `::device::meater` in `device_meater.tcl` or something similar?

This `msg -DEBUG` makes sense for early development, but probably needs a flag to disable or to be removed later on. 


Passing in `event_time`, even if unused, means that later processing can know when the packet arrived for processing. Depending on how much is inline code of the handlers from previous packets, it can be delayed by 100-250 ms or more. I didn't convert all of the existing calls in `bluetooth.tcl` due to the number of changes. Right now, it's on a packet-by-packet basis, but I can definitely see puling events off the `ble` callback and onto the event queue if I can't get potentially slow things like `skins_page_change_due_to_de1_state_change` out of the BLE handler and onto the appropriate queue.

_(Reminds me to try to get that onto an event. When I last tried, the GUI failed to render. Hmmm, now it works.)_

_(and you ruin a coffee because the timestamp fetch is done blocking and locks up the machine for 30 seconds)_

In `bluetooth.tcl`:
```
   1194 proc de1_ble_handler { event data } {
   1195         #msg "de1 ble_handler '$event' [convert_string_to_hex $data]"
   1196         #set ::de1(wrote) 0
   1197 
   1198         set event_time [expr { [clock milliseconds] / 1000.0 }]
```

In `device_scale.tcl`:
```
    193         proc process_weight_update {reported_weight {event_time 0}} {
    194 
    195                 if { $event_time == 0 } {set event_time [expr { [clock milliseconds] / 1000.0 }]}
```